index.js

```diff
@@ -185,6 +185,11 @@ module.exports = function(options) {
     fn.metricsQueue = []
     const config = setConfig(options)
 
+    if (! config.clientId) {
```

Comment:
I'd also recommend removing this check here, as this change would make it extraneous: https://github.com/ewindisch/iopipe-agent-node/blob/1b3bb448904af5d8527ed4bfd9e7e196550432a3/index.js#L148